.circleci/config.yml

```diff
@@ -211,6 +211,8 @@ references:
         echo ${TESTS}
         bundle exec rspec ${TESTS}
         #COVERAGE=1 bundle exec rspec
+      environment:
```

Comment:
rather than add this variable to CircleCI, how about adding it to the existing file `.env.test` - that way it can be used for local tests as well as CircleCI. Also, where did the value `//` come from? maybe a slightly more conventional would be clearer, e.g.:
```
SENTRY_DSN='http://something@example.test/'
```